Bad error message for when compressing zip archive fails

### Version

bda6c270629711cd9072826471553c58766daf56 (main branch)

### Description

When compressing a Zip archive fails, the error message is terrible.


### Current Behavior

```
$ ouch compress /tmp archive.zip
Do you want to overwrite 'asd.zip'? [Y/n] 
[ERROR] No such device or address (os error 6)
```

### Expected Behavior

Should give the reason why IO failed, like compressing to `Tar` does:

```
$ ouch compress /tmp archive.tar
[ERROR] failed to open file `tmp/scoped_dirbOXtNO/SingletonSocket`
```

### Additional Information

_No response_

--- a/src/archive/zip.rs
+++ b/src/archive/zip.rs
@@ -4,7 +4,6 @@
 use std::os::unix::fs::PermissionsExt;
 use std::{
     env,
-    fs::File,
     io::{self, prelude::*},
     path::{Path, PathBuf},
     sync::mpsc,
@@ -223,7 +222,7 @@ where
                     options
                 };
 
-                let mut file = File::open(entry.path())?;
+                let mut file = fs::File::open(entry.path())?;
                 writer.start_file(
                     path.to_str().unwrap(),
                     options.last_modified_time(get_last_modified_time(&file)),
@@ -256,7 +255,7 @@ fn display_zip_comment_if_exists(file: &ZipFile) {
     }
 }
 
-fn get_last_modified_time(file: &File) -> DateTime {
+fn get_last_modified_time(file: &fs::File) -> DateTime {
     file.metadata()
         .and_then(|metadata| metadata.modified())
         .map_err(|_| ())

